packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -150,7 +155,7 @@ const styles = StyleSheet.create({
     },
 
     smallColumn: {
-        flex: "0 0",
+        flex: "1",
```

Comment:
After getting the TwoColumnModal to render it was looking really bad, this fixed it, mostly. There are still some other issues with how it displays but I'll file those as follow-up issues.